test/common/wsServerMaster.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var ChildProcess = require('child_process');
+var path = require('path');
+var Promise = require('bluebird');
+var node = require('../node.js');
+var WSClient = require('./wsClient');
+
+/**
+ * WSServerMaster
+ * Create a socket server master instance with the support to spawn multiple processes
+ *
+ * @constructor
+ */
+function WSServerMaster () {
+	this.masterProcess = null;
+	this.client = null;
+	this.port = Math.floor(Math.random() * 9999);
+}
+
+/**
+ * Start the socket server instance. It will start the server as well an instance of the client.
```

Comment:
Start the socket server master instance. It will start the server and an instance of the client.

Reply:
Done.